backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchConfiguration.java

```diff
@@ -218,6 +230,10 @@ public int getMaxRetries() {
         return maxRetries;
     }
 
+    public Duration getRequestTimeout() {
```

Comment:
No modification of `ElasticSearchConfiguration::fromProperties(...)`  ? Wouldn't this be useful?